android/src/main/java/com/google/samples/apps/iosched/droidconitaly15/JsonTransformer.java

```diff
@@ -248,6 +262,19 @@ private static void transformJson(
         sessions.add(session);
     }
 
+    /**
+     * MOAR COLORZ PLS!
+     */
+    private static String getColorFrom(String roomName) {
+        String roomColor = DEFAULT_ROOM_COLOR;
```

Comment:
Nah no point in lowercasing the room names, they always come through the same, and we're hacking stuff here :P